k8s/secrets.staging.enc.yaml

```diff
@@ -1,90 +1,91 @@
-apiVersion: ENC[AES256_GCM,data:17o=,iv:DqkDV/AR/NcKNDVPHKj4lyGxIrvcR5p3zYKlHPZtZRg=,tag:d/SllcsYkK7NegeTph3hKg==,type:str]
```

Comment:
I think we'll have to re-edit this file starting from current master.  This file got changed in master meanwhile and the merge story for sops isn't very good (actually, there isn't really one).  And I'll also have to make similar edits to the production secrets file to add the correct cross domain for production deployment.

Reply:
I merged master onto this branch, but this file still has different data.